compact/ecal/barrel_interlayers_birks.xml

```diff
@@ -0,0 +1,205 @@
+<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
+<!-- Copyright (C) 2022 Whitney Armstrong, Chao Peng, Maria Zurek, Jihee Kim -->
+
```

Comment:
Now we move on to the birks file, which only adds a new material and uses it instead, but keeps the rest of the structure of `barrel_interlayers.xml` unchanged. Again, this makes sense to put in a separate file, which now would contain:
```xml
<lccdd>
  <define>
     <constant name="EcalBarrelImagingLayers_nMax"   value="6"/>
     <constant name="EcalBarrelImagingLayers_FiberMaterial" value="SciFiPb_Scintillator_Birks"/>
  </define>
</lccdd>
```
The definition of the material itself should be placed in the overall materials file, and likely be given a name which includes the actual Birks value (it's only a matter of time before someone wants to try a differen value), so maybe `SciFiPb_Scintillator_Birks0079`.